app/src/components/main/attributeList/editableAttribute/attributeEditor/attributeEditor.jsx

```diff
@@ -173,6 +173,7 @@ export class AttributeEditor extends Component {
             />
           </FieldErrorHint>
         </div>
+        <span className={cx('separator')}>:</span>
```

Comment:
Now this separator applying for all places where we are using attribute editor. Does it expected behavior?